README.md

```diff
@@ -1,168 +1,187 @@
+Wouldn't it be nice if page refreshes didn't exist? What if we could do multiple things at once from a single web page? In a perfect world we could type into a search textbox and have searches being performed in the background as we type. That world is here and it's called Ajax! Asynchronous Javascript and XML (Ajax) is a technique that is used in web applications. It provides a way for content to be retrieved from a server and displayed without having to refresh the entire page.
 
-Single page applications provide a better user experience by allowing users to manipulate data on the server and see the results without having to refresh the page.  This is the power of Ajax in action.  In the background, requests are made to a web API using JavaScript.  As developers we can then choose to alter the displayed HTML based on the responses from the web API.
+Modern dynamic applications provide a better user experience by allowing users to manipulate data on the server and see the results without having to refresh the page. This is the power of Ajax in action. In the background, requests are made to a web API using JavaScript. As developers we can then choose to alter the displayed HTML based on the responses from the web API.
 
-(Like the GitHub API gem, Ajax is a _library_ for making HTTP requests. The big difference between a gem that makes HTTP requests and a JavaScript library that does the same is that the former makes the requests synchronously, whereas the latter makes the requests asynchronously — one consequence of this difference is that, with JavaScript, that even though the requests still _fire_ in order, they might return out of order.)
+When you've used `XMLHttpRequest` directly to query an API like Github
+and dynamically update your page, you've been doing Ajax. With jQuery,
+we have a set of Ajax-related functions to make that even easier.
 
-Let's try an example. To start, create a folder on your computer called `example`. Inside that folder create another folder called `html`. Now create a file named `index.html`, `script.js` and `html/sentence.html`. Finally add the following to each file.
+Let's try an example. You have an `index.html` file with a basic structure. Let's add a reference to jQuery so we can use it.
 
-#### index.html
 ```html
-<!doctype html>
-<head>
-  <script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jquery/1.3/jquery.min.js"></script>
-    <script src="script.js"></script>
-</head>
 <body>
   <p id="sentences">
     Loading our page...
   </p>
+  <script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jquery/1.3/jquery.min.js"></script>
```

Comment:
hahaha I'm dumb. I updated it in the code in the HTML page, and then of course deleted all that code so they could code along.